src/translationService.ts

```diff
@@ -0,0 +1,61 @@
+/* This simple app uses the '/translate' resource to translate text from
+one language to another. */
+
+import uuid4 from 'uuid4';
+import dotenv from 'dotenv';
+import axios, { AxiosError, AxiosResponse } from 'axios';
+import { TranslatedLanguage } from './types/translation-api-response';
+
+dotenv.config();
+
+const resourceKey = process.env.TRANSLATOR_TEXT_RESOURCE_KEY;
+if (!resourceKey) {
+  throw new Error('Please set/export the following environment variable: TRANSLATOR_TEXT_RESOURCE_KEY');
+}
+
+const endpoint = process.env.TRANSLATOR_TEXT_ENDPOINT;
+if (!endpoint) {
+  throw new Error('Please set/export the following environment variable: TRANSLATOR_TEXT_ENDPOINT');
+}
+
+const region = process.env.TRANSLATOR_TEXT_REGION;
+if (!region) {
+  throw new Error('Please set/export the following environment variable: TRANSLATOR_TEXT_REGION');
+}
+
+/* If you encounter any issues with the base_url or path, make sure that you are
+using the latest endpoint: https://docs.microsoft.com/azure/cognitive-services/translator/reference/v3-0-translate */
+// export async function translateText(lang, value): Promise<Array<Record<string, unknown>>> {
+export async function translateText(lang, value): Promise<unknown> {
```

Comment:
If you set the type of the params, example:
```
translateText(lang: string, value: string)
```
then you can remove the `lang as string` and `value as string` below.

Also, I believe the return value is a string. So that can be updated to:
```
Promise<string> 
```